pkg/recovery/permissiverecoverer.go

```diff
@@ -0,0 +1,37 @@
+package recovery
+
+import (
+	"log"
+	"os/exec"
+)
+
+const shell string = "rush"
+
+// PermissiveRecoverer properties
+// Debug: Enables recovery shell
+type PermissiveRecoverer struct {
+	Debug bool
+}
+
+// Recover logs error message in panic mode.
+// Can jump into a shell for later debugging.
+func (pr PermissiveRecoverer) Recover(message string) error {
```

Comment:
this interface is a bit misleading: it makes the user thing they have to provide a message, but if the message is an empty string it will behave differently. I would rather just print a message (not a panic) and run the shell if specified

Reply:
Done